src/main/views/waveform.zoomview.js

```diff
@@ -80,7 +80,10 @@ define([
 
           // enable drag if necessary
           var mouseMove = function(event) {
-            peaks.seeking = false;
+			var pixelsDifference = Math.abs(event.layerX - x);
```

Comment:
Scratch that I was looking at looking at MDN documentation but there is the possibility that this is not the case with the way chrome deals with high resolution screens